.github/actions/label-released-issues-v1/src/getReferencedClosedIssues.ts

```diff
@@ -0,0 +1,91 @@
+import { getOctokit } from '@actions/github'
+
+interface GetReferencedClosedIssuesArgs {
+  owner: string
+  repo: string
+  pullRequestID: number
+  octokit: ReturnType<typeof getOctokit>
+}
+
+/**
+ * GraphQL query response:
+ * @example
+ * ```json
+ * {
+ *  "data": {
+ *    "repository": {
+ *      "pullRequest": {
+ *        "closingIssuesReferences": {
+ *          "nodes": [
+ *            {
+ *              "number": 27
+ *            }
+ *          ]
+ *        }
+ *      }
+ *    }
+ *   }
+ * }
+ * ```
+ */
+export interface GetReferencedClosedIssuesResult {
+  repository: {
+    pullRequest: {
+      closingIssuesReferences: {
+        nodes: {
+          number: number
+          repository: {
+            owner: { login: string }
+            name: string
+          }
+        }[]
+      }
+    }
+  }
+}
+
+export default async function getReferencedClosedIssues({
```

Comment:
No need to duplicate this, we can utilise this from https://github.com/dequelabs/axe-api-team-public/blob/6af842cd18148974e6eb05a40b0328fa7bcfbb7f/.github/actions/label-and-move-released-issues-v1/src/getReferencedClosedIssues.ts#L47